packages/theme/package.json

```diff
@@ -3,14 +3,23 @@
   "version": "3.6.1",
   "private": false,
   "description": "Utilities building and publishing themes",
+  "homepage": "https://github.com/shopify/cli#readme",
+  "bugs": {
+    "url": "https://github.com/Shopify/cli/issues"
+  },
   "type": "module",
+  "license": "MIT",
   "files": [
     "/dist"
   ],
+  "publishConfig": {
+    "access": "public",
+    "@shopify:registry": "https://registry.npmjs.org"
+  },
   "scripts": {
     "clean": "shx rm -rf dist",
     "build": "rimraf dist/ && tsc -b ./tsconfig.build.json",
-    "prepack": "cross-env NODE_ENV=production yarn run build",
+    "prepack": "cross-env NODE_ENV=production yarn run build && cp ../../README.md README.md",
```

Comment:
Nice trick 😝

Reply:
I did it in a few other packages as well.